plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -45,32 +45,68 @@ jQuery( function( $ ) {
 			$tabs_wrapper.find( '#' + $tab.attr( 'href' ).split( '#' )[1] ).show();
 		} )
 		.on( 'keydown', '.wc-tabs li a, ul.tabs li a', function( e ) {
+			var isRTL     = document.documentElement.dir === 'rtl';
 			var direction = e.key;
-			var next      = 'ArrowRight';
-			var prev      = 'ArrowLeft';
+			var right     = isRTL ? 'ArrowLeft' : 'ArrowRight';
+			var left      = isRTL ? 'ArrowRight' : 'ArrowLeft';
```

Comment:
Nitpick: but I think it would be better to leave `next` and `prev` as the variable names now? I feel `right` being `ArrowLeft` might be confusing.

Reply:
@Aljullu super valid nitpicking, I agree! Reverted to the next/prev variable names, thanks!